packages/daemon/src/generator/tvm.ts

```diff
@@ -0,0 +1,16 @@
+import { fork } from 'child_process';
+import { GenerateOptions } from '../services/interface';
+import * as path from 'path';
+
+export function generateTVM(dist: string, projPackage: any, opts: GenerateOptions) {
+  return new Promise((resolve, reject) => {
+    const client = fork(`${path.resolve(__dirname, 'tvm.cli')}`, [ JSON.stringify({
+      dist,
+      projPackage,
+      opts
+    }), 'keras' ]);
+
+    client.on('message', () => resolve({}));
```

Comment:
Again, we need to handle the binary/JSON to ensure this is complete correctly :)